TestAssets/TestProjects/TestLibraryWithMultipleFrameworks/Program.cs

```diff
@@ -7,6 +8,10 @@ public class Program
         public static void Main()
         {
             Console.WriteLine("Hello World!");
+#if !NETSTANDARDAPP1_5
```

Comment:
Is this going to bite us as we go to 1.6? Is this really just for 3.5? Can we condition on that?